Src/WindowsServer/WindowsServer.Shared/Implementation/AzureRoleEnvironmentContextReader.cs

```diff
@@ -62,25 +86,76 @@ internal set
         /// </summary>
         internal static string BaseDirectory { get; set; }
 
+        internal static string AssemblyName { get; set; }
```

Comment:
We don't know what the fully qualified name of assembly will be yet - so whats the advantage of wrapping the string inside AssemblyName class?